core-plugins/src/e2e-test/resources/testdata/file/expected_outputs/OUTPUT_FOR_TSV.csv

```diff
@@ -0,0 +1,30 @@
+TS-GCS-DSGN-01-09,To verify the browse button functionality by selecting the path of the file through browse button,TC-GCS-DSGN-01,positive
+TS-GCS-DSGN-01-10,To verify the output of get schema by using browse button functionality,TC-GCS-DSGN-10,positive
+TS-GCS-DSGN-01-11,To verify the output of get schema without using browse button functionality, direcltly place the path,TC-GCS-DSGN-11,Negative
+TS-GCS-DSGN-01-12,Verify the GCS properties of Delimeter field if the file format is delimeted,TC-GCS-DSGN-12,positive
+TS-GCS-DSGN-01-13,Verify the GCS properties of Overrride field by placing the column and data type to be override, data type supported:  Bytes double float  Int long string date time timestamp boolean,TC-GCS-DSGN-13,positive
+TS-GCS-DSGN-01-14,Verify the various combinations in override field:  Int to double double to float  Int to long Int to boolean ,TC-GCS-DSGN-14,Positive
+TS-GCS-DSGN-01-15,To verify the invalid format of data types in override field,TC-GCS-DSGN-15,Negative
+TS-GCS-DSGN-01-01,To verify Get schema button to validate output schema,TC-GCS-DSGN-16,positive
+TS-GCS-DSGN-01-01,To verify Skip header button with true value and validate the schema that header should skipped,TC-GCS-DSGN-17,positive
+TS-GCS-DSGN-01-01,To verify Skip header button with false value and validate the schema that header should not be skipped,TC-GCS-DSGN-18,negative
+TS-GCS-DSGN-01-01,In the credentials verify service account type field by selecting filepath radio button,TC-GCS-DSGN-19,positive
+TS-GCS-DSGN-01-01,In the credentials verify service account type field by selecting JSON radio button,TC-GCS-DSGN-20,positive
+TS-GCS-DSGN-01-01,To verify Regex path filter by validating regular expression for files to read,TC-GCS-DSGN-21,positive
+TS-GCS-DSGN-01-01,In path field put the path of the file and verify that if output field to contain the path of the object that was read from,TC-GCS-DSGN-22,positive
+TS-GCS-DSGN-01-01,Verify the path field name only field with true value to check whether to use file name only instead of URI,TC-GCS-DSGN-23,positive
+TS-GCS-DSGN-01-01,Verify the GCS properties of Read files recursively field with true or false value,TC-GCS-DSGN-24,positive
+TS-GCS-DSGN-01-01,Verify the GCS properties of Data file encrypted field with true or false value,TC-GCS-DSGN-25,positive
+TS-GCS-DSGN-01-01,Verify the GCS properties of encryption metadata file suffix field,TC-GCS-DSGN-26,positive
+TS-GCS-DSGN-01-01,Verify the GCS properties of File system properties field which is Additional properties to use with the InputFormat when reading the data,TC-GCS-DSGN-27,positive
+TS-GCS-DSGN-01-01,Verify the GCS properties of File encoding field(UTF-8),TC-GCS-DSGN-28,positive
+TS-GCS-DSGN-01-01,Verify the maximum split size for each partion in bytes in Advance GCS properties,TC-GCS-DSGN-29,positive
+TS-GCS-DSGN-01-01,Verify the minimum split size for each partion in bytes in Advance GCS properties,TC-GCS-DSGN-30,positive
```

Comment:
Do we need test data with so many rows??

Reply:
No not required. Please suggest maximum row count. 10 ?

Reply:
Reduced rows.